spec/lib/terraform/runner_spec.rb

```diff
@@ -352,4 +352,60 @@ def verify_req(req)
       end
     end
   end
+
+  context '.parse_template_variables hello-world' do
+    describe '.parse_template_variables input/output vars' do
+      template_variables_stub = nil
+
+      def verify_req(req)
+        body = JSON.parse(req.body)
+        expect(body).to(have_key('templateZipFile'))
+      end
+
+      before do
+        ENV["TERRAFORM_RUNNER_URL"] = "https://1.2.3.4:7000"
```

Comment:
This can cause issues if other specs need different values here in the future, because the before is setting a real ENV var but nothing is resetting it.

This is where `stub_const` helps, e.g.:
`before { stub_const("ENV", ENV.to_h.merge("CONTAINER_IMAGE_NAMESPACE" => "registry.example.com/manageiq")) }`

Reply:
:-| frankly did not know, you could stub ENV var this way in ruby ...
 
 Thanks, will update the code accordingly

Reply:
:tada: for learning something new